### ISSUE TITLE
cannot specify virtual network when deploying APIM usnig CLI

### ISSUE THREAD

**Issue by vlad610** [Azure/azure-cli#24830]
The documentation is not clear on how a VNet integrated APIM service can be created.

I tried this:

_az login
az apim create --name "vlad33" --publisher-email "vlad.dancescu@microsoft.com" --publisher-name "vlad d" --resource-group "vlad1" --virtual-network "External"_

The service created is not VNet integrated.

The section for "az apim create", specifically, the "--virtual-network" parameter needs to be better documented. There is no option to specify the virtual network and subnet you want to deploy the service into.

This was raised here as well:
https://stackoverflow.com/questions/65058942/azure-api-management-how-to-set-the-vnet-subnet-using-azure-cli

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 197d8382-ee16-6001-4941-768877379c0d
* Version Independent ID: 1423fa76-977f-ad88-8129-0b52a38e68e6
* Content: [az apim](https://learn.microsoft.com/en-us/cli/azure/apim?view=azure-cli-latest)
* Content Source: [latest/docs-ref-autogen/apim.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/latest/docs-ref-autogen/apim.yml)
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
route to CXP team

**Comment by SaurabhSharma-MSFT** [comment]
@vlad610 You are correct and so redirecting this issues to respective team to look into.
